layouts/_default/staff-page.html

```diff
@@ -212,86 +207,94 @@ <h3 class="inline-block self-start leading-snug">
         {{ end }}
       </h2>
       <p class="text-sm font-black uppercase">{{ $author.title }}</p>
-      <dl class="mt-3 flex flex-wrap">
+      <dl>
         {{ with $author.email }}
-          <dt class="flex-0 mr-2 flex items-center">
-            <span class=" mr-2 h-4 w-4">
-              <svg viewBox="0 0 1 1">
-                <use xlink:href="#email-svg"></use>
-              </svg>
-            </span>
-            Email:
-          </dt>
-          <dd class="flex-[0_1_50%]">
-            <a
-              class=" text-blue"
-              data-ga-category="author-email"
-              href="mailto:{{ . }}"
-              title="Email"
-            >
-              {{ . }}
-            </a>
-          </dd>
+          <div class="flex basis-0 items-start justify-between gap-2">
```

Comment:
I don't like wrapping the definitions in a div, but I couldn't keep them from sometimes wrapping wrong without it.